libvmi/driver/kvm/kvm_events.c

```diff
@@ -170,10 +170,6 @@ process_cb_response(
         errprint("%s: invalid kvm or kvmi handles\n", __func__);
         return VMI_FAILURE;
     }
-    // Note: libvmi_event can be NULL
```

Comment:
Correct me if I'm wrong but afais this comment is not accurate because `events_destroy` is called after `driver_destroy`.

Reply:
That's correct, I reviewed the code and `events_destroy()` is called after, so the ghashtables should contain the libvmi events